utils/vmd_utils.py

```diff
@@ -1,9 +1,10 @@
 import re
 import unicodedata
 import simplejson as jsoncode
+from unidecode import unidecode 
 
 def urlify(s):
     s = re.sub(r"[^\w\s\-]", '', s)
     s = re.sub(r"\s+", '-', s).lower()
-    return ''.join(c for c in unicodedata.normalize('NFD', s) if unicodedata.category(c) != 'Mn')
+    return ''.join(c for c in unidecode(unicodedata.normalize('NFD', s)) if unicodedata.category(c) != 'Mn')
```

Comment:
Je pense que vu la complexité du problème, éviter de réinventer l'eau chaude c'est pas mal. `unidecode` fait très bien le boulot, mais il n'y a plus besoin de `unicodedata` : 
```suggestion
    return unidecode(s)
```
Par contre, `unidecode` ne fait pas partie de la bibliothèque standard (c'est pour ça que je voulais l'éviter initialement), il va falloir le télécharger sur PyPI & co. Il faut que je vois comment on fait ça.